client/src/main/scala/drt/client/services/SPACircuit.scala

```diff
@@ -27,29 +27,51 @@ sealed trait ViewMode {
 
   def dayStart: SDateLike = SDate.midnightOf(time)
 
+  def dayEnd: SDateLike = dayStart.addDays(1).addMinutes(-1)
+
   def time: SDateLike
 
-  def isHistoric: Boolean
+  def isLive: Boolean
+
+  def isHistoric(now: SDateLike): Boolean
 }
 
 case object ViewLive extends ViewMode {
   def time: SDateLike = SDate.now()
 
-  def isHistoric: Boolean = false
+  override val isLive: Boolean = true
+
+  override def isHistoric(now: SDateLike): Boolean = false
```

Comment:
this parameter isn't used, so you could replace with `_`

Reply:
i thought so too but unfortunately you can't replace with a `_`